chapter_codecs.md

```diff
@@ -4,8 +4,32 @@ title: Chapter Codecs
 
 # Matroska Chapter Codecs
 
+Chapter codecs are way to add more complex playback features than the usual linear playback.
+
+Some `ChapProcess` elements hold commands to execute when entering/leaving a chapter.
+
+When chapter codecs are uses the `EditionFlagOrdered` of the edition they belong to **MUST** be set.
```

Comment:
Google trasnslater suggested to use "used" instead of "uses".

> When chapter codecs are used the `EditionFlagOrdered` of the edition they belong to **MUST** be set.